puppet/zulip/files/supervisor/conf.d/zulip.conf

```diff
@@ -7,40 +7,20 @@
 ; Note: shell expansion ("~" or "$HOME") is not supported.  Environment
 ; variables can be expanded using this syntax: "%(ENV_HOME)s".
 
-[fcgi-program:zulip-django]
-command=/home/zulip/deployments/current/manage.py runfcgi daemonize=False maxchildren=20 ; the program (relative uses PATH, can take args)
-;process_name=%(program_name)s ; process_name expr (default %(program_name)s)
-;numprocs=1                    ; number of processes copies to start (def 1)
-;directory=/tmp                ; directory to cwd to before exec (def no cwd)
-;umask=022                     ; umask for process (default None)
+[program:zulip-django]
+command=/home/zulip/deployments/current/zulip-venv/bin/uwsgi --ini /etc/zulip/uwsgi.ini
```

Comment:
This seems to be related to Ubuntu version. I checked manually in ubuntu 14.04 and it looks like that uWSGI works with Python over there. (I am not sure at the moment though).